### PR TITLE
Add definitions for querying balances with v2 datanode api

### LINKED ISSUE #286: Add proto definitions for querying balances with v2 datanode API

### PR DESCRIPTION
**Pull request by pscott31**
Closes #286

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
 - [272](https://github.com/vegaprotocol/protos/pull/272) - Update snapshot protos to be able to handle the iavl exported nodes
 - [276](https://github.com/vegaprotocol/protos/pull/276) - Remove maturity from future product, tick size from market and trading mode config from market
 - [277](https://github.com/vegaprotocol/protos/pull/277) - Update validation for risk params for log normal risk model
+- [287](https://github.com/vegaprotocol/protos/pull/287) - Add `QueryBalance` call and associated messages
```
Comment (emilbayes) [inline CHANGELOG.md:53]: I think the name here needs updating :)

File: sources/data-node/api/v2/trading_data.proto [hunk 2]
```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+option go_package = "code.vegaprotocol.io/protos/data-node/api/v2";
+
+import "vega/vega.proto";
+import "google/protobuf/wrappers.proto";
+
+package datanode.api.v2;
+
+service TradingDataService {
```
Comment (emilbayes) [inline sources/data-node/api/v2/trading_data.proto:10]: Will this be the final service name or do we want to start doing smaller, categorized services?

File: CHANGELOG.md (hunk 1 above)
Comment (pgs31) [reply CHANGELOG.md]: Thanks, done

File: sources/data-node/api/v2/trading_data.proto (hunk 2 above)
Comment (pgs31) [reply sources/data-node/api/v2/trading_data.proto:10]: It's up for discussion I guess but I'm not sure it's worth the extra faff, I'm going to commit it for now :)

Comment (jeremyletang) [reply sources/data-node/api/v2/trading_data.proto:10]: I think Emil suggestion is good. let's try to follow the guidelines he's defined.

**Pull request merged**